restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -1102,6 +1102,23 @@ private String getParentPathFromRow(Row row) {
     return s.toString();
   }
 
+  private boolean pathsMatch(String path1, String path2) {
```

Comment:
If both paths will be empty it will return true. Is it expected?

Reply:
Yea that's fine; in practice both paths should never really be empty but that's reasonable behavior in that case